environments/env_generator/examples/ketchup_mustard_hotdog.json

```diff
@@ -0,0 +1,82 @@
+{
+    "version": "1.0.0",
+    "width": 3,
+    "height": 3,
+    "config": {
+        "cook_time": {
+            "hotdog": 3,
+            "default": 3
+        }
+    },
+    "stations": [
+        {
+            "name": "table",
+            "x": 0,
+            "y": 1
+        },
+        {
+            "name": "grill",
+            "x": 2,
+            "y": 1,
+            "id": "A"
+        },
+        {
+            "name": "table",
+            "x": 0,
+            "y": 0
+        },
+        {
+            "name": "table",
+            "x": 2,
+            "y": 0
+        }
+    ],
+    "items": [
+        {
+            "name": "hotdog",
+            "x": 0,
+            "y": 1,
+            "stack-level": 0,
+            "predicates": ["isgrillable"],
+            "id": "a"
+        }
+    ],
+    "condiments": [
+        {
+            "name": "ketchupbottle",
+            "x": 0,
+            "y": 0,
+            "stack-level": 0,
+            "predicates": ["isketchup"],
+            "id": "b"
+        },
+        {
+            "name": "mustardbottle",
+            "x": 2,
+            "y": 0,
+            "stack-level": 0,
+            "predicates": ["ismustard"]
+        }
+    ],
+    "players": [
+        {
+            "name": "robot",
+            "x": 1,
+            "y": 0,
+            "direction": [-1, 0]
+        }
+    ],
+    "goal description": "Cook a hot dog and add ketchup and mustard on it.",
+    "goal": [
+        {
+            "predicate": "has_ketchup_mustard",
```

Comment:
no need to check for this predicate, simply check if it `has_ketchup` and also `has_mustard`

Reply:
replaced with those two predicates